src/Tests/packages.config

```diff
@@ -5,8 +5,8 @@
   <package id="Mono.Cecil" version="0.10.1" targetFramework="net452" />
   <package id="Newtonsoft.Json" version="11.0.2" targetFramework="net452" />
   <package id="NServiceBus" version="6.2.0" targetFramework="net452" />
-  <package id="NUnit" version="3.10.1" targetFramework="net452" />
-  <package id="NUnit3TestAdapter" version="3.12.0" targetFramework="net452" />
+  <package id="NUnit" version="3.12.0" targetFramework="net452" />
+  <package id="NUnit3TestAdapter" version="3.13.0" targetFramework="net452" />
   <package id="Particular.Approvals" version="0.1.0" targetFramework="net452" />
```

Comment:
You should go ahead and update this to the latest as well since there's no longer a reason to stay on 0.1.0.